### ISSUE TITLE
USER ISSUE: tools disappear after letting go of cart

### ISSUE THREAD

**Issue by PattyPlays** [StrangeLoopGames/EcoIssues#20134]
\
**Version:** 0.9.1.9 beta release-136

**Steps to Reproduce:**
1. equip a tool (f.e. an axe)
2. use a cart
3. let go of the cart
(this goes for all tools)

**Expected behavior:**
tool that was in hand before should be visible like normal

**Actual behavior:**
tool has disappeared from view. hands are empty, but in inventory slots it looks like it is still equipped and i can still chop trees with bare hands. 

**Do you have mods installed? Does the issue happen when no mods are installed?:**
no.

**Please provide log files. Instruction is here https://github.com/StrangeLoopGames/EcoIssues/wiki/Getting-the-Eco-game-log-file

**Comment by SlayksWood** [comment]
should be in 3d person view

**Comment by PattyPlays** [comment]
> should be in 3d person view

Yes that's right! In 3rd person view the tools disappear from hand.
